### PR TITLE
Creating chart

### PR DESCRIPTION
**Pull request by quandang246**
## Related Tickets
- [ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/69212)

## WHAT (optional)
- Creating chart for homepage. 

## HOW
- i add chart.js file, edit homepage.html

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Review by riderhuy — APPROVED** [review]
Approve

**Review by tdanhjv — APPROVED** [review]

**Comment by quandang246** [comment]
ready

File: air_quality_management/myapp/templates/base.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
     integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
     crossorigin="anonymous">
     <link rel="stylesheet" href="{% static 'css/base.css' %}" />
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
```
Comment (huongnt-2545) [inline air_quality_management/myapp/templates/base.html]: tải source về project nhé e ơi

**Comment by quandang246** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**